sdk/core/azure-core/CHANGELOG.md

```diff
@@ -8,29 +8,46 @@
 - Added `Range` type to `Azure::Core::Http` namespace.
 - Added support for long-running operations with `Operation<T>`.
 - Added support for setting a custom transport adapter by implementing the method `std::shared_ptr<HttpTransport> ::AzureSdkGetCustomHttpTransport()`.
+- Added interoperability between `std::chrono::system_clock` types and `DateTime`.
 - Added default constructor to `DateTime`.
 - Added `DateTime` supporting dates since 0001.
 - Added Base64 encoding and decoding utility APIs to the `Azure::Core` namespace available from `azure/core/base64.hpp`.
 - Added `Http::Response<void>` template specialization.
+- Added `GetHeadersAsString()` on the `Azure::Core::Http::Request` class.
+- Added a `platform.hpp` header file for defining commonly used OS-specific `#define` constants.
+- Added `IsCancelled()` on the `Azure::Core::Context` class.
 
 ### Breaking Changes
 
-- Removed `DateTime::operator Duration()`.
+- Removed `DateTime::operator Duration()` and `DateTime::Duration` typedef.
 - Removed `DateTime::Now()`.
 - Moved `Azure::Core::BearerTokenAuthenticationPolicy`, defined in `azure/core/credentials.hpp` to `Azure::Core::Http` namespace in `azure/core/http/policy.hpp` header.
-- Removed option `AllowBeast` from `CurlTransportSSLOptions` in `CurlTransportOptions`.
-- Changed default option `NoRevoke` from `CurlTransportSSLOptions` for the `CurlTransportOptions` to `true`. This disables the revocation list checking by default.
 - Changed type of `Token::ExpiresOn` to `DateTime`.
 - Renamed exception `OperationCanceledException` to `OperationCancelledException`.
-- Renamed methods from `Azure::Core::Context`.
-  - `IsCanceled` to `IsCancelled`
-  - `ThrowIfCanceled` to `ThrowIfCancelled`.
+- Renamed `ThrowIfCanceled()` to `ThrowIfCancelled()` on the `Azure::Core::Context` class.
 - Moved `Azure::Core::Version`, defined in `azure/core/version.hpp` to the `Azure::Core::Details` namespace.
+- Changed `Azure::Core::AuthenticationException` to derive from `std::exception` instead of `std::runtime_error`.
+- Changed the `BodyStream::Read` API from being a pure virtual function to non-virtual.
+- Removed `CurlConnection`, `CurlConnectionPool`, `CurlSession` and `CurlNetworkConnection` by making these headers private:
+  - curl_connection.hpp
+  - curl_connection_pool.hpp
+  - curl_session.hpp
```

Comment:
Does this cover everything @vhvb1989? Can you confirm just in case I missed something from this PR (https://github.com/Azure/azure-sdk-for-cpp/pull/1023) or others that might be related?

Reply:
FWIW we've had some feedback that our changelogs in C / C++ are too verbose.  Focus on highlevel change descriptions and reference back to the git changes for additional details when necessary

Reply:
it's perfect. Thank you